index.html

```diff
@@ -33,9 +33,10 @@
 					<li class="strawberry">Strawberry</li>
 				</ul>
 				<div class="results">
-					<h2 id="title"></h2>
-					<p id="description"></p>
+					<h2 id="title">Apple</h2>
```

Comment:
The idea here was that you would use jQuery to target the element with an id of "title" and an id of "description" and dynamically update the html via JavaScript.

e.g.

```js
$('#title').html('Apple');
$('#description').html('The apple tree (Malus pumila, commonly and erroneously called Malus domestica)...');
```